cahiers-d-ethnomusicologie.csl

```diff
@@ -0,0 +1,484 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="fr-FR">
+  <info>
+    <title>Cahiers d'ethnomusicologie (French)</title>
+    <id>http://www.zotero.org/styles/cahiers-d-ethnomusicologie</id>
+    <issn>1662-372X</issn>
+    <eissn>2235-7688</eissn>
+    <link
+	href="http://www.zotero.org/styles/cahiers-d-ethnomusicologie"
+	rel="self"/>
+    <link href="http://www.zotero.org/styles/iso690-author-date-en"
+	  rel="template" />
+    <link
+	href="http://www.adem-geneve.com/fr/cahiers-d-ethnomusicologie-contribution"
+	rel="documentation" />
+    <author>
+      <name>Victor A. Stoichita</name>
+      <email>svictor@svictor.net</email>
+      <uri>http://www.svictor.net</uri>
+    </author>
+    <category citation-format="author-date"/>
```

Comment:
add a field from the lists [here](http://docs.citationstyles.org/en/stable/specification.html#appendix-i-categories). I'd think `<category field="humanities"/>` fits best.